src/IO/Observer/VolumeActions.cpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "IO/Observer/VolumeActions.hpp"
+
+#include <string>
+#include <vector>
+
+#include "DataStructures/Tensor/TensorData.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/Observer/ObservationId.hpp"
+
+namespace observers::ThreadedActions::VolumeActions_detail {
+void write_data(const std::string& h5_file_name,
+                const std::string& subfile_path,
+                const observers::ObservationId& observation_id,
+                std::vector<ElementVolumeData>&& volume_data) {
+  const uint32_t version_number = 4;
```

Comment:
why version 4? At least a C++ comment would be helpful... Because normally I start at version 0 or 1...

Reply:
There's not a good reason, so I will siwtch this to 0. I thought our volume data version was 4, but I misunderstood that nonzero version numbers were only used for testing (Test_H5.cpp, e.g.).